.github/workflows/test_frameworks.yml

```diff
@@ -94,6 +94,43 @@ jobs:
         # django.tests.requests module interferes with requests library patching in the tracer -> disable requests patch
         run: DD_TRACE_REQUESTS_ENABLED=0 ddtrace-run tests/runtests.py
 
+  graphene-testsuite-3_0:
+    runs-on: ubuntu-latest
+    env:
+      DD_PROFILING_ENABLED: true
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: graphene
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v3
+        with:
+          repository: graphql-python/graphene
+          ref: v3.0.0
+          path: graphene
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
+      - name: Install ddtrace
+        run: pip install ../ddtrace
+      - name: Install graphene
+        run: pip install -e "../graphene[test]"
+      - name: "Upgrade pytest"
+        # pytest<6.2.5 raises `TypeError: required field "lineno" missing from alias` on python 3.10
+        run: pip install "pytest>=6.2.5,<7"
+      - name: Set Pythonpath
+        run: echo "PYTHONPATH=." >> $GITHUB_ENV
+      - name: Run tests
+        # The following tests are broken and will be skipped in this framework test
```

Comment:
Are these broken upstream?

Reply:
Yup, here's an example: https://github.com/graphql-python/graphene/runs/5619442650?check_suite_focus=true. CI should be unblocked in the next release. Here's the fix: [PR](https://github.com/graphql-python/graphene/pull/1412)